symfony/plugins/orangehrmAdminPlugin/lib/dao/I18NDao.php

```diff
@@ -22,21 +22,21 @@ class I18NDao extends BaseDao
     /**
      * @param string $langCode
      * @param bool $onlyCustomized
-     * @return Doctrine_Collection|I18NTranslate[]
+     * @return I18NTranslate[]
      * @throws DaoException
      */
     public function getMessages(string $langCode, bool $onlyCustomized = true)
     {
         try {
-            $q = Doctrine_Query::create()
-                ->from('I18NTranslate t')
-                ->leftJoin('t.I18NLangString ls')
-                ->leftJoin('t.I18NLanguage l')
-                ->andWhere('l.code = ?', $langCode);
+            $q = Doctrine2::getEntityManager()->getRepository(I18NTranslate::class)->createQueryBuilder('t');
```

Comment:
This part may be abstract to DAO class .. something like $this->getQueryBuilder()